src/vector-crypto.adoc

```diff
@@ -308,8 +313,13 @@ all other `SEW` values are _reserved_.
 
 |===
 
-Source/Destination overlap constraints::
-Some Vector Crypto instructions have overlap constraints. Encodings that violate these constraints are _reserved_.
+Vector/Scalar constraints::
+This specification defines new vector/scalar (.vs) instructions that used *Scalar ELement Groups*. The *Scalar Element Group* operand has `EMUL = ceil(EGW / VLEN)`.
```

Comment:
Replace "used" with "use"
This specification defines new vector/scalar (.vs) instructions that _use_ *Scalar ELement Groups*.

Reply:
```suggestion
This specification defines new vector/scalar (.vs) instructions that uses *Scalar Element Groups*. The *Scalar Element Group* operand has `EMUL = ceil(EGW / VLEN)`.
```